pandas/core/ops.py

```diff
@@ -604,40 +496,298 @@ def _get_op_name(op, special):
 """
 
 _flex_doc_FRAME = """
-{desc} of dataframe and other, element-wise (binary operator `{op_name}`).
+{desc} of dataframe and other, element-wise (binary operator `{op_name}`
+and reverse version `{reverse}`).
 
-Equivalent to ``{equiv}``, but with support to substitute a fill_value for
-missing data in one of the inputs.
+Among flexible wrappers (`add`, `sub`, `mul`, `div`, `mod`, `pow`) to
+arithmetic operators.
+
+Equivalent to `+`, `-`, `*`, `/`, `//`, `%`, **` but with support to
+substitute a fill_value for missing data in one of the inputs.
 
 Parameters
 ----------
-other : Series, DataFrame, or constant
-axis : {{0, 1, 'index', 'columns'}}
-    For Series input, axis to match Series index on
-level : int or name
+other : scalar, sequence, Series, or DataFrame
+    Any single or multiple element data structure, or list-like object.
+axis :  {{0 or 'index', 1 or 'columns'}}
+    Whether to compare by the index (0 or 'index') or columns
+    (1 or 'columns'). For Series input, axis to match Series index on.
+level : int or label
     Broadcast across a level, matching Index values on the
-    passed MultiIndex level
-fill_value : None or float value, default None
+    passed MultiIndex level.
+fill_value : float or None, default None
     Fill existing missing (NaN) values, and any new element needed for
     successful DataFrame alignment, with this value before computation.
     If data in both corresponding DataFrame locations is missing
-    the result will be missing
+    the result will be missing.
 
 Notes
 -----
-Mismatched indices will be unioned together
+Mismatched indices will be unioned together.
 
 Returns
 -------
-result : DataFrame
+DataFrame
+    Result of the arithmetic operation.
+
+See Also
+--------
+DataFrame.add : Add DataFrames.
+DataFrame.sub : Subtract DataFrames.
+DataFrame.mul : Multiply DataFrames.
+DataFrame.div : Divide DataFrames (float division).
+DataFrame.truediv : Divide DataFrames (float division).
+DataFrame.floordiv : Divide DataFrames (integer division).
+DataFrame.mod : Calculate modulo (remainder after division) of
+    DataFrames.
+DataFrame.pow : Calculate exponential power of DataFrames.
 
 Examples
 --------
-{df_examples}
+>>> df = pd.DataFrame({{'A': [4, 6, 8],
+...                    'B': [3, 5, 9]}})
```

Comment:
The indentation here doesn't seem right.

Also, I think it'd be better to have a simple real-world example, so it's easier to follow. For example, in the next example is quite obvious to understand what the operator is doing, even without checking the original DataFrame. With "random" data things are more complex for the user.
```
>>> df + 1
           num_legs  num_wings
cat          5        1
ostrich      3        3
tuna         1        1
```
Do you mind changing it?